EXPECTED_PATTERN[1004] in test/parser/let.error.2.pytch, line 1, character 4:
Error: I was expecting a pattern after 'let'.
   +-----------------------------------------------------+
   | test/parser/let.error.2.pytch                       |
 1 | let                                                 |
   |    ^ Error: I was expecting a pattern after 'let'.  |
   | ^~~ Note: This is the beginning of the let-binding. |
 2 | let foo = bar                                       |
   +-----------------------------------------------------+
EXPECTED_PATTERN[1004] in test/parser/let.error.2.pytch, line 3, character 4:
Error: I was expecting a pattern after 'let'.
   +-----------------------------------------------------+
   | test/parser/let.error.2.pytch                       |
 2 | let foo = bar                                       |
 3 | let                                                 |
   |    ^ Error: I was expecting a pattern after 'let'.  |
   | ^~~ Note: This is the beginning of the let-binding. |
   +-----------------------------------------------------+
